<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="b_apply">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>550</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Appliquer</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_filters">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>510</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="b_load_img">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>540</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Charger</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_load">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>510</y>
      <width>241</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>rs.pgm</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_im_ori">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>360</width>
      <height>360</height>
     </rect>
    </property>
    <property name="text">
     <string>image initiale</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_im_end">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>360</width>
      <height>360</height>
     </rect>
    </property>
    <property name="text">
     <string>image finale</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
